Jenkinsfile

```diff
@@ -0,0 +1,64 @@
+pipeline {
+    agent {
+        dockerfile {
+        }
+    }
+    stages {
+        stage('Setup') {
+            steps {
+                sh 'npm --version'
+                sh 'pwd'
+                sh 'env'
+                sh 'ls -al'
+            }
+        }
+        stage('Build') {
+            steps {
+                sh 'make all'
+            }
+        }
+        stage('Upload - Development') {
+            when {
+                environment name: 'RCH_ENVIRONMENT', value: 'development'
+            }
+            steps {
+                withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'jenkins-aws-development', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
```

Comment:
This allows us to NOT have to map a volume to .aws and root users.

I have added the creds to Jenkins.

Reply:
Is there any way to restrict access to this in Jenkins? i.e. we don't want to have just anyone log into jenkins and see these creds?

Reply:
believe the creds are encrypted once entered into the credentials. so no chance of viewing.